cranelift/filetests/filetests/isa/aarch64/nan-canonicalization.clif

```diff
@@ -21,7 +21,7 @@ block0(v0: f32x4, v1: f32x4):
 ;   movdqa %xmm0, %xmm2
 ;   movdqa %xmm1, %xmm0
 ;   movdqa %xmm2, %xmm1
-;   pblendvb %xmm1, %xmm7, %xmm1
+;   pblendvb %xmm0, %xmm7, %xmm1
```

Comment:
I'll note that there's no actual register allocation changes here, this is just an artifact of how the instruction used to be printed. The xmm0 argument was omitted by default and the src/destination, which are required to be the same register, were printed twice.